NEWS

```diff
@@ -1,11 +1,9 @@
-x/y/2015 ola-0.9.6
+30/6/2015 ola-0.9.6
  Features:
- * Default to using epoll() and kqueue() rather than select() if they exist,
-   apart from for the HTTP thread.
+ * Default to using epoll() rather than select() if it exists,
```

Comment:
I turned it off until we can get the serial port problems fixed.

I don't think the HTTP stuff is relevant for the NEWS